src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.logo_link {
+  height: 40px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__menu {
+  display: flex;
+  list-style: none;
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
CSS: Be consistent with your vertical margins. It's a good practice to add only top or only bottom margin, but not both. This is to avoid issues with margin collapsing.